### PR TITLE
Adding varz for replication unfriendly statements

### PR DESCRIPTION
**Pull request by erzel**
@michael-berlin please take a look. Thanks.

### CODE REVIEW

File: go/vt/sqlannotation/sqlannotation.go [hunk 1]
```diff
@@ -25,6 +27,10 @@ const (
 	deleteDML                            = "delete"
 )
 
+var (
+	replicationUnfriendlyCountStat = stats.NewInt("replication-unfriendly-statements-count")
```
Comment (aaijazi) [inline go/vt/sqlannotation/sqlannotation.go]: I believe open-source convention is that we use UpperCamelCase. We do break it a couple of times - and I realize the example I pointed you to was wrong, sorry!

**Comment by aaijazi** [comment]
You have a bunch of commits for a small change here. I think it would be a good idea to squash your commits into one. Something like: https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

File: go/vt/sqlannotation/sqlannotation.go (hunk 1 above)
Comment (michael-berlin) [inline go/vt/sqlannotation/sqlannotation.go]: Sugu won't like this but I would go even further and name the variable ```filteredReplicationUnfriendlyCount``` (or ```...CountStat```) to be clear that this is not MySQL replication but filtered replication instead.
(You already have ```filteredReplicationUnfriendlyComment``` in the code :)

Just for reference for myself: Ammar is talking about the name of the stats variable, not the variable name itself.

Comment (sougou) [inline go/vt/sqlannotation/sqlannotation.go]: I just saw this review :)
Variable names have to be camel cased. Also, /debug/vars produces JSON, which means that you can have hierarchical values. They get translated correctly on import.

**Comment by erzel** [comment]
@aaijazi  -- I tried to follow the steps in the doc you linked. I'm not sure it worked. Should I do anything different?

**Comment by erzel** [comment]
I have renamed both the statz var and the go var as @michael-berlin  suggested. PTAL.

**Comment by michael-berlin** [comment]
LGTM

Thanks. Can you please also squash your commits? :-)

After the squash, you need to force push your branch to Github. Let me know if you need help.

**Pull request merged**